frontend/src/app/reducers/browser.js

```diff
@@ -1,9 +1,12 @@
 import { handleActions } from 'redux-actions';
 
 const setIsFirefox = (state, { payload: isFirefox }) => ({ ...state, isFirefox });
+const setIsMinFirefox = (state, { payload: isMinFirefox }) => ({ ...state, isMinFirefox });
 
 export default handleActions({
-  setIsFirefox
+  setIsFirefox,
+  setIsMinFirefox
 }, {
-  isFirefox: false
+  isFirefox: false,
+  isMinFirefox: false
```

Comment:
The version of Firefox will never update while the application is running.

Rather than go through the full action => reducer process, just set the value in default props here.